src/util/misc.js

```diff
@@ -607,31 +607,19 @@
      * @memberOf fabric.util
      * @param {Array} elements SVG elements to group
      * @param {Object} [options] Options object
-     * @param {String} path Value to set sourcePath to
      * @return {fabric.Object|fabric.Group}
      */
-    groupSVGElements: function(elements, options, path) {
-      var object;
+    groupSVGElements: function(elements, options) {
       if (elements && elements.length === 1) {
         return elements[0];
       }
       if (options) {
-        if (options.width && options.height) {
-          options.centerPoint = {
-            x: options.width / 2,
-            y: options.height / 2
-          };
-        }
-        else {
+        if (!options.width || !options.height) {
           delete options.width;
           delete options.height;
         }
       }
-      object = new fabric.Group(elements, options);
-      if (typeof path !== 'undefined') {
-        object.sourcePath = path;
-      }
-      return object;
+      return new fabric.Group(elements, options);
```

Comment:
```suggestion
      return new fabric.Group(elements, Object.assign(options || {}, { layout: 'svg' }));
```

You missed this

Reply:
So the issue with the SVG is that the group creation code is perfectly able to mantain position and size, but then the position was changed from the layout strategy.
Removing options entirely works, but of course it does not work for SVG that are outside the bounding box.

Reply:
I still see an difference in outputs margin but it doesn't surface in the visual tests